specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/stable/2017-04-18/cognitiveservices.json

```diff
@@ -124,12 +124,11 @@
             "description": "The name of Cognitive Services account."
           },
           {
-            "name": "parameters",
+		    "name": "account",
```

Comment:
breaking change...

Reply:
Hi Kirs, this is just a name in swagger. The request body doesn't change.